contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -370,6 +378,202 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         }
     }
 
+    /**
+     * @notice Executes a flashLoan operation with the requested assets.
+     * @dev Transfer the specified assets to the receiver contract and handles repayment.
+     * @param initiator The address of the EOA who initiated the flash loan.
+     * @param receiver The address of the contract that will receive the flashLoan amount and execute the operation.
+     * @param vTokens The addresses of the vToken assets to be loaned.
+     * @param underlyingAmounts The amounts of each underlying assets to be loaned.
+     * @param param The bytes passed in the executeOperation call.
+     * @custom:error FlashLoanNotEnabled is thrown if the flash loan is not enabled for the asset.
+     * @custom:error InvalidAmount is thrown if the requested amount is zero.
+     * @custom:error NoAssetsRequested is thrown if no assets are requested for the flash loan.
+     * @custom:error InvalidFlashLoanParams is thrown if the flash loan params are invalid.
+     * @custom:error SenderNotAuthorizedForFlashLoan is thrown if the sender is not authorized to use flashloan.
+     * @custom:event Emits FlashLoanExecuted on success
+     */
+    function executeFlashLoan(
```

Comment:
Would it make sense to move this function (and the related ones) to a new facet: `FlashLoanFacet`? That would not increase the size of the `PolicyFacet`